### PR TITLE
fix: Group issues should also group names

### PR DESCRIPTION
**Pull request by mladkau**
- [x] Ready for review
- [x] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/.github/CONTRIBUTING.md) rules
- [x] Reviewed by Snyk internal team

#### What does this PR do?

Small fix for the `--group-issues` flag. We should also group the name attribute along with the from attribute.

#### Screenshots

image

### CODE REVIEW

File: src/cli/index.ts [hunk 1]
```diff
@@ -254,7 +254,11 @@ async function main() {
         '--group-issues is currently not supported for Snyk IaC.',
       ]);
     }
-    if (args.options['group-issues'] && !args.options['json']) {
+    if (
+      args.options['group-issues'] &&
+      !args.options['json'] &&
+      !args.options['json-file-output']
```
Comment (mladkau) [inline src/cli/index.ts:260]: Not strictly part of the PR but the `json-file-output` should also be allowed with the `group-issues` flag.

File: test/fixtures/basic-apk/jsonDataGrouped.json [hunk 2]
```diff
@@ -103,7 +103,21 @@
       "upgradePath": [],
       "isUpgradable": false,
       "isPatchable": false,
-      "name": "musl/musl-utils",
+      "name": [
```
Comment (JackuB) [inline test/fixtures/basic-apk/jsonDataGrouped.json:106]: Should we also make these unique? Or the array length matters?

**Review by JackuB — APPROVED** [review]
Small question

File: test/fixtures/basic-apk/jsonDataGrouped.json (hunk 2 above)
Comment (mladkau) [reply test/fixtures/basic-apk/jsonDataGrouped.json:106]: I would leave the array as is as each entry can be matched with an entry in the from list.

**Pull request merged**